script.js

```diff
@@ -75,30 +75,29 @@ const game = (event) => {
       playable = false;
       if (cardsTurned[0].innerHTML === cardsTurned[1].innerHTML) {
         const turnCardsOut = () => {
-          for (let i = 0; i < cardsTurned.length; i++) {
-            cardsTurned[i].classList.remove("card-turned");
-            cardsTurned[i].classList.add("card-out");
-          }
+          cardsTurned.forEach((card) => {
+            card.classList.remove("card-turned");
+            card.classList.add("card-out");
+          });
           cardsTurned = [];
           playable = true;
           gameOver();
         };
         setTimeout(turnCardsOut, 500);
       } else {
+        const turnCardsBack = () => {
+          cardsTurned.forEach((card) => {
+            card.classList.remove("card-turned");
+          });
+          cardsTurned = [];
+          playable = true;
+        };
         setTimeout(turnCardsBack, 500);
```

Comment:
Tady by se mi líbilo u funkcí `turnCardsOut` a `turnCardsBack`
* buď arrow funkci neukládat do proměnné, ale strčit ji celou do `setTimeout` přímo
* a nebo přesunout obě funkce úplně ven, tam, co byla původně jen funkce `turnCardsBack` (obě funkce nepotřebují žádnou lokální proměnnou ani žádný parametr a není potřeba je při každém kliknutí definovat znovu)